### DISCUSSION TITLE
Feature List for first release - 0.0.1

### DISCUSSION THREAD

**Discussion by firozemzr** [OSBDAlliance/headlesscms#15] [Show and tell]
For the first release I am planning to include

1. Basic content Add, Update, Delete and Retrieve with version control
2. A common logging structure
3. A common Authentication, Authorization and Accounting structure for all service/content delivery
4. Database Agnostic Implementation
5. Lifecycle/Release Management (Update of new version of headless CMS in existing deployment)

**Comment by openbravo** [comment]
Please check the User Stories and Tasks. https://github.com/OSBDAlliance/headlesscms/issues. @firozemzr 
We will share with the tech team and update the user stories and tasks if needed.
